Formula/gnutls.rb

```diff
@@ -33,14 +31,10 @@ class Gnutls < Formula
   depends_on "p11-kit"
   depends_on "unbound"
 
-  on_linux do
-    depends_on "autogen"
-  end
-
   def install
-    # Fix build with Xcode 12
-    # https://gitlab.com/gnutls/gnutls/-/issues/1116
-    ENV.append "CFLAGS", "-Wno-implicit-function-declaration"
+    # Fix compile crash on Apple Silicon.
+    # https://gitlab.com/gnutls/gnutls/-/issues/1347
+    inreplace "lib/accelerated/aarch64/Makefile.in", /^(AM_CCASFLAGS = -Wa),-march=all$/, "\\1" if OS.mac?
```

Comment:
Seems weird to retain the `-Wa` flag still if you're no longer passing anything to the assembler.

Granted, that's what the upstream patch does, but still weird.

Reply:
Good point, I'm not sure why it's like that - I'll do it properly here. It's not fully upstreamed yet so probably hasn't been through proper review.